tests/publiccloud/create_aistack_env.pm

```diff
@@ -210,7 +245,7 @@ sub test_openwebui_service {
 
 
 sub install_nvidia_drivers {
-    my ($instance, $values_url, $file_name) = @_;
+    my ($instance, $file_name) = @_;
     my $easyinstall_url = 'https://gitlab.nue.suse.com/cloud-solutions-sys-eng/nvidia-drivers-easy-install/-/blob/main/nvidia_easy_install.sh';
     my $driver_version = '550.54.14';
     my $gpu_op_url = get_var('GPU_OPERATOR');
```

Comment:
if this value is mandatory, then use `get_required_val(...`

Reply:
this is temporary solution..very soon install_nvidia_drivers will be replace with a easy_install shell script which will be re-purpose for customer use also.